<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeakEditDlg</class>
 <widget class="QDialog" name="PeakEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Peak Edit - ATREX</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Peak # :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_id">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_hkl_0">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_hkl_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_hkl_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>hkl :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>detXY :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_detXY_0">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_detXY_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_xyz_0">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_xyz_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_xyz_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xyz :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>d-spc :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_dspace">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Energy :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_energy">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_gon_0">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_gon_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_gon_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_gon_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_gon_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_gon_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gonio :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Intensity :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_intens">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Image :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pe_SourceImageLE">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
